cli/command/image/build.go

```diff
@@ -293,6 +293,9 @@ func runBuild(dockerCli *command.DockerCli, options buildOptions) error {
 
 	response, err := dockerCli.Client().ImageBuild(ctx, body, buildOptions)
 	if err != nil {
+		if options.quiet {
```

Comment:
The change is to adjust the case in `TestBuildNotVerboseFailure` because it requires stderr output in quiet mode and stdout+stderr output in verbose mode to be equal. Without the change the `Sending build context to Docker daemon 2.048 kB` will be missing from the output:

```
func (s *DockerSuite) TestBuildNotVerboseFailure(c *check.C) {
        // This test makes sure that -q works correctly when build fails by
        // comparing between the stderr output in quiet mode and in stdout
        // and stderr output in verbose mode
        tt := []struct {
                TestName  string
                BuildCmds string
        }{
                {"quiet_build_no_from_at_the_beginning", "RUN whoami"},
                {"quiet_build_unknown_instr", "FROMD busybox"},
        }

        for _, te := range tt {
                _, _, qstderr, qerr := buildImageWithStdoutStderr(te.TestName, te.BuildCmds, false, "-q", "--force-rm", "--rm")
                _, vstdout, vstderr, verr := buildImageWithStdoutStderr(te.TestName, te.BuildCmds, false, "--force-rm", "--rm")
                if verr == nil || qerr == nil {
                        c.Fatal(fmt.Errorf("Test [%s] expected to fail but didn't", te.TestName))
                }
                if qstderr != vstdout+vstderr {
                        c.Fatal(fmt.Errorf("Test[%s] expected that quiet stderr and verbose stdout are equal; quiet [%v], verbose [%v]", te.TestName, qstderr, vstdout+vstderr))
                }
    }
```